[TECH] Montée de version de Pix UI sur Admin

yannbertrand opened the pull request:
## :jack_o_lantern: Problème
**Passage en v15**

Les fonts google étaient encore chargées sur Pix Admin alors que Pix UI les intègre désormais.

**Passage en v16**

Certains nom d'icônes fontawesome ont été remplacés.

## :bat: Proposition
Supprimer le chargement des google fonts sur Pix Admin.

Changement des alias d'icônes fontawesome.

## :spider_web: Remarques
Entre la version 15.0.0 et 15.0.1, on force une version de npm stricte qui fait casser la pipeline car ce n'est pas la version de nos applis fronts. J'ai poussé au delà de la v15.0.1 pour bénéficier de https://github.com/1024pix/pix-ui/pull/237

## :ghost: Pour tester
Sur la page de connexion de pix admin, vérifier qu'aucun appel a googleapis.com est fait.

Vérifier sur les différents écrans que les icônes s'affichent bien.

yannbertrand committed:
chore(admin): remove ember-cli-google-fonts

pix-service commented:
I'm deploying this PR to these urls:

- App (.fr): https://app-pr5145.review.pix.fr
- App (.org): https://app-pr5145.review.pix.org
- Orga: https://orga-pr5145.review.pix.fr
- Certif: https://certif-pr5145.review.pix.fr
- Admin: https://admin-pr5145.review.pix.fr
- API: https://api-pr5145.review.pix.fr/api/

 Please check it out!

yannbertrand committed:
chore(admin): upgrade pix-ui to 15.0.0

PixMultiSelect is only used one time (target profile creation form) and seems to still work

yannbertrand committed:
chore(admin): upgrade pix-ui to v16.1.0

Updated some icons names, hopefully I catch them all.

AndreiaPena commented on admin/tests/integration/components/routes/authenticated/target-profiles/target-profile/old-details_test.js:
```diff
@@ -95,7 +95,7 @@ module('Integration | Component | routes/authenticated/target-profiles/target-pr
       // then
       assert.dom('[data-icon="check"]').exists({ count: 2 });
-      assert.dom('[data-icon="times"]').exists({ count: 14 });
+      assert.dom('[data-icon="xmark"]').exists({ count: 14 });
```
je pleure de voir ça mais c'est pas le but de la PR

AndreiaPena commented:
J'ai parcouru plusieurs pages, pas vu d'icônes cassée.

Sur le PixMultiSelect de la création de profil cible, j'ai fait un comparatif avec dev et il y a une différence de largeur visuellement, mais le reste est bon.
avant : 
<img width="695" alt="Capture d’écran 2022-10-28 à 18 03 08" src="https://user-images.githubusercontent.com/58915422/198682683-30f96fa6-7f01-46dc-9d56-ab9921d02548.png">


après :
<img width="695" alt="Capture d’écran 2022-10-28 à 18 03 13" src="https://user-images.githubusercontent.com/58915422/198682691-a6678090-aaff-4496-bc26-03174cbc5506.png">

AndreiaPena reviewed (approved):
good pour moi !

francois2metz reviewed (approved):
:leafy_green: :maple_leaf: :four_leaf_clover: :fallen_leaf:  LGTM :fallen_leaf:  :four_leaf_clover: :maple_leaf: :leafy_green:

nlepage reviewed (approved):
LGTM
